content/docs/search/reactivesearch-api/APIReference.md

```diff
@@ -130,6 +141,16 @@ For example:
         "unit": "mi/yd/ft/km/m/cm/mm/nmi"
     }
 ```
+
+The below example represents the `geo` value format for a geo bounding box query:
+```js
```

Comment:
@bietkul Show a complete example that can be copy/pasted v/s just the specific key.